inc/wpseo-non-ajax-functions.php

```diff
@@ -573,3 +358,61 @@ function wpseo_allow_system_file_edit() {
 
 	return WPSEO_Utils::allow_system_file_edit();
 }
+
+/**
+ * Test whether force rewrite should be enabled or not.
+ *
+ * @deprecated 3.3
+ *
+ * @return void
+ */
+function wpseo_title_test() {
+	return;
```

Comment:
Just showing intent to have an empty function not that something 'has been forgotten'.